e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -137,4 +137,27 @@ describe('Message', () => {
             cy.get(divPostId).should('have.class', 'post--system');
         });
     });
+
+    it('message with emoji contains hidden shortcode text', () => {
+        // # Post a message with a shortcode emoji
+        cy.postMessage('This post has a shortcode emoji :raising_hand_man: within it.');
+
+        cy.getLastPostId().then((postId) => {
+            const divPostId = `#post_${postId}`;
+
+            cy.get(divPostId).find('span.emoticon').should('have.text', ':raising_hand_man:');
```

Comment:
I'm good with this but please add one more assertion:
```javascript
cy.get(`#postMessageText_${postId}`).should('have.text', message);
```

Reply:
Added the assertion @saturninoabril